tasklist/client/e2e/pages/FormJSDetailsPage.ts

```diff
@@ -73,5 +94,29 @@ class FormJSDetailsPage {
       await this.page.getByText(value, {exact: true}).click();
     }
   }
+
+  getLocatorByLabel(label: string) {
+    return this.page.getByLabel(label);
+  }
+
+  getLocatorByText(label: string) {
+    return this.page.getByText(label);
+  }
```

Comment:
Just use `this.page.getByText`, there's no need to create an extra abstraction over it